docs/product-portal/accounts/dev-process.md

```diff
@@ -271,56 +180,8 @@ Here are some handy questions and things to consider when reviewing code for Fir
 
 ## Tagging releases
 
-Each repo has a `grunt` script
-for tagging new releases.
-This script is responsible for:
-
-* Updating the version strings
-  in `package.json` and `npm-shrinkwrap.json`.
-
-* Writing commit summaries
-  to the change log.
-
-* Committing these changes.
-
-The script will not push the tag,
-so you can always check what's changed
-before making the decision
-about whether the changes were correct
-and it's okay to push.
-
-To tag a major release, run:
-
-```
-grunt version
-```
-
-To tag a patch release, run:
-
-```
-grunt version:patch
-```
-
-Patch releases should normally be tagged
-in a specific `train-nnn` branch,
-which must then be merged back to `master`.
-
-It's important that:
-
-1. The merge happens;
-
-2. It really is just a vanilla `git merge`
-   and not a `rebase`, `cherry-pick` or `merge --squash`.
+The release process is being [documented and reviewed](https://docs.google.com/document/d/1lAK-Wvmr7xPsVHIDVFhG5RlQKQQL1Z4NlhyCPLLq4-U/edit?ts=5cb8ba2d).  We'll update this official documentation when the details are sorted out.
```

Comment:
Users have to unfortunately sign into GDocs to view this.

Reply:
That's ok for now.  Once we finalize it we'll add it to the real docs